sentinelsat/scripts/cli.py

```diff
@@ -237,13 +237,13 @@ def cli(
             search_kwargs["area"] = geojson_to_wkt(read_geojson(geometry))
         # check if the value is a GeoJSON
         else:
-            geometry = geometry.strip()
-            if geometry[0] == "{":
+            if geometry.startswith("{"):
                 try:
                     geometry = json.loads(geometry)
                     search_kwargs["area"] = geojson_to_wkt(geometry)
-                except SyntaxError:
-                    logger.info("geometry string starts with '{' but is not a valid GeoJson.")
+                except JSONDecodeError:
+                    logger.error("geometry string starts with '{' but is not a valid GeoJSON.")
+                    exit(1)
```

Comment:
Just a quick comment: I think we should avoid `print ; exit`, even in the interface, and instead use high-level ways of terminating. Here, perhaps `raise click.UsageError` would be correct?

Reply:
Thanks for the input. It felt kind of hacky to me but I wasn't sure how to fix it.